graph/src/data/subgraph/mod.rs

```diff
@@ -614,6 +630,15 @@ impl UnresolvedMapping {
     }
 }
 
+async fn cat_module(
+    resolver: &impl LinkResolver,
+    logger: &Logger,
+    link: &Link,
+) -> Result<Arc<Module>, failure::Error> {
+    let module_bytes = resolver.cat(logger, link).await?;
+    Ok(Arc::new(parity_wasm::deserialize_buffer(&module_bytes)?))
+}
+
```

Comment:
Hah, this function has more signature/boilerplate code than actual code. 😁 It's still fine, just made me giggle.

Reply:
I think I can fix this. I didn't know about `async move` yet when I wrote it.